### PR TITLE
Doc TiDB/TiCDC

### LINKED ISSUE #762: Integration/Use case: TiDB/TiCDC
### Related code PR

https://github.com/risingwavelabs/risingwave/pull/8708

### Which part(s) of the docs might be affected or should be updated? And how?

Eric created a demo for ingesting CDC data from TiDB to RW. Let's try out the demo and convert it into a use case or tutorial.

Update [**Integrations**](https://www.risingwave.dev/docs/current/rw-integration-summary/): TiDB source available (https://www.risingwave.dev/docs/current/create-source-cdc/); sink available. Add a link to the new use case ⬇️.

Add a new use case in **Use cases** to demonstrate that users can use RisingWave to build a real-time computation pipeline for their database tables. A materialized view, effectively.

Add TiDB to **Sinks**. Or update [**Sinks - MySQL**](https://www.risingwave.dev/docs/current/sink-to-mysql-with-jdbc/) to include TiDB. To be discussed with @fuyufjh.

### Reference

_No response_

### PR DESCRIPTION
**Pull request by emile-00**
## Info
- **Description**: 
Add topic on sinking to TiDB. Add on to TiCDC description in CDC via event streaming systems topic. 

- **Preview**: 
[ Paste the preview link to the edited page here. ]

- **Related code PR**: 
https://github.com/risingwavelabs/risingwave/pull/8708

- **Related doc issue**: 
Resolves https://github.com/risingwavelabs/risingwave-docs/issues/762

- **Notes**: 
[ Any additional information? ]

## Before merging
  - [ ] (For version-specific PR) I have selected the corresponding software version in **Milestone** and linked the related doc issue to this PR in **Development**.
  - [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`bernscode`, `CharlieSYH`, `emile-00`, & `hengm3467`). 
  - [ ] I have checked the doc site preview, and the updated parts look good. <details><summary>How?</summary>Scroll down and open this link: image
</details>

### CODE REVIEW

File: docs/create-source/create-source-cdc.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ RisingWave accepts these data formats:
 
 - The TiCDC dialect of Canal JSON (for TiDB only)
 
-  For the TiCDC dialect of [Canal](https://github.com/alibaba/canal) JSON (`ROW FORMAT CANAL_JSON`), you can add TiCDC to an existing TiDB cluster to convert TiDB data changes to Kafka topics. For details, see [Deploy and Maintain TiCDC](https://docs.pingcap.com/tidb/dev/deploy-ticdc). 
+  For the TiCDC dialect of [Canal](https://github.com/alibaba/canal) JSON (`ROW FORMAT CANAL_JSON`), you can add TiCDC to an existing TiDB cluster to convert TiDB data changes to Kafka topics. You might need to define the topic name in a TiCDC config file. Note that for TiCDC only captures new changes from TiDB. Data that already exists within the target table will not be captured by TiCDC. For details, see [Deploy and Maintain TiCDC](https://docs.pingcap.com/tidb/dev/deploy-ticdc).
```
Comment (hengm3467) [inline docs/create-source/create-source-cdc.md]: ..Note that for TiCDC... "for" seems redundant.

File: sidebars.js [hunk 2]
```diff
@@ -295,6 +295,11 @@ const sidebars = {
               label: 'MySQL',
               id: 'guides/sink-to-mysql-with-jdbc'
             },
+            {
```
Comment (hengm3467) [inline sidebars.js]: @emile-00 We might want to place TiDB below PosgreSQL (if we want to order either by popularity or alphabetically).

**Review by hengm3467 — APPROVED** [review]
Otherwise looks good to me. Thanks. @emile-00

**Review by bernscode — APPROVED** [review]

**Review by fuyufjh — APPROVED** [review]
LGTM

**Comment by CharlieSYH** [comment]
Do we need to update the [Integrations](https://www.risingwave.dev/docs/current/rw-integration-summary/) page? @emile-00 
image

**Pull request merged**